### ISSUE TITLE
AttributeError: 'Plot' object has no attribute 'datetimes_to_string'

### ISSUE THREAD

**Issue by froggleston** [freqtrade/ftui#9]
Since update 5.3 of plotext, the `datetimes_to_string` function is now named `datetimes_to_strings`. A quick solution is to pin `plotext==5.2.8` but really this needs to be updated properly.

Raised with textual-plotext in https://github.com/Textualize/textual-plotext/issues/12

**Comment by froggleston** [comment]
Should be addressed in https://github.com/freqtrade/ftui/commit/d5b066402ccb09bdf6cba1a71c23166f69a8e364

**Comment by tdv78** [comment]
Same problem

**Comment by froggleston** [comment]
Yes, I need to do a release. In the meantime, you can use the main branch code and it will work fine.

**Comment by tdv78** [comment]
Thank you!
